karma/karma-tests.ts

```diff
@@ -28,6 +28,12 @@ karma.runner.run({port: 9876}, (exitCode: number) => {
   process.exit(exitCode);
 });
 
+karma.stopper.stop({port: 9876}, function(exitCode) {
+  if (exitCode === 0) {
+    console.log('Server stop as initiated')
```

Comment:
No, it is from [the karma website](http://karma-runner.github.io/1.0/dev/public-api.html). I think it's like: 

> Server stopped, because you initiated it.

Or something